controller/trackerquery_blackbox_test.go

```diff
@@ -182,7 +183,17 @@ func TestCreateTrackerQueryREST(t *testing.T) {
 	app.MountTrackerqueryController(service, controller2)
 
 	server := httptest.NewServer(service.Mux)
-	tqPayload := fmt.Sprintf(`{"query": "abcdefgh", "schedule": "1 1 * * * *", "trackerID": "%s"}`, tracker.ID)
+	tqPayload := fmt.Sprintf(`{
+		"query": "abcdefgh",
+		"schedule": "1 1 * * * *",
+		"trackerID": "%s",
```

Comment:
They are just three input arguments :)... I can understand that if I have n > 5. Anyway I will check that